src/include/sof/trace/trace.h

```diff
@@ -305,7 +305,7 @@ struct tr_ctx {
  * @param default_log_level Default log level.
  */
```

Comment:
@ktrzcinx Please be more generous with the explanations in the commit message. 

It is not very clear which entries will be more easier to iterate over. Also, is there a typo here? 'over entries of ..'  ->  'over entries OR'

Reply:
Wording seems a bit obtuse. I'd rephrase to "It is the first step to allow iterating over the UUID (or trace context?) entries and searching for the right one."

Reply:
updated